### DISCUSSION TITLE
Invalid Invocation on findOne

### DISCUSSION THREAD

**Discussion by Tenkir** [prisma/prisma#2302] [Q&A] [answered]
Having some issues with findOne. Have the following query:
```
const brand = await prisma.brand.findOne({
  where: { id: args.id }
});
```

But get back Invalid Invocation when trying to run it. In my schema I have:
```
model Brand {
  id  Int  @id @default(autoincrement())
  ...
}
```
As well, the error message in GraphQL Playground is cluttered with unicode chacters and is completely unreadable. Any way to remove them?

<img width="1630" alt="image" src="https://user-images.githubusercontent.com/3649828/80288945-707e4400-8709-11ea-9f69-0a6f54fed7bd.png">

**Comment by Tenkir** [comment, marked as answer]
Silly mistake. Was passing the ID as a String, not as an Int.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
